velox/exec/tests/TableWriteTest.cpp

```diff
@@ -610,3 +575,122 @@ TEST_F(TableWriteTest, writeNoFile) {
   execute(plan, std::make_shared<core::QueryCtx>(executor_.get()));
   ASSERT_TRUE(fs::is_empty(outputDirectory->path));
 }
+
+TEST_F(TableWriteTest, appendToUnpartitionedTableWithExistingTableType) {
```

Comment:
I'm still confused about these tests. I see 4 new tests:

- appendToUnpartitionedTableWithExistingTableType
- createUnpartitionedTableAndinsertWithNewTableType
- appendToUnpartitionedTable
- appendToUnpartitionedTableSequence

I would assume appendToUnpartitionedTableWithExistingTableType tests appending data to an existing unpartitioned table. In this case, I'd expect it to be named appendToUnpartitionedTable. Then, I wonder what does appendToUnpartitionedTable do and how it is different from appendToUnpartitionedTableWithExistingTableType.

I would then assume that createUnpartitionedTableAndinsertWithNewTableType tests creating new unpartitioned table using some data. This use case should have been already supported. Did we not have a test for it? If so, I'd expect it to be named createUnpartitionedTable.

Is there any particular reason there are 4 tests and not just 2: one for creating unpartitioned table and another for appending to an existing table?

Reply:
I refactored the tests and removed the redundant test cases so it is three tests now:
- Test that we can create and insert to a new table regardless of the value of the config.
- Test that we are not allowed to write to an existing table when the config is enabled.
- Test that we can write to an existing table when the config is disabled.